README.md

```diff
@@ -1,17 +1,17 @@
-# Long Form Audio Testing
-This project provides a ready-to-use template for doing in-depth testing on long-form audio.
+# **Long Form Audio Testing**
```

Comment:
I think we should add a table-of-contents section under the main header like this:
<h1 align="center">Long Form Audio Testing</h1>
<p align="center">Setup | Running | GitLab | DataDog </p>